biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -164,7 +165,8 @@ The Business Activity Monitoring Tools include:
 - BAM Portal
 
 
-- Configuring BAM tools requires certain SQL Server administrative functionality and must be performed from a machine that has Integration Services installed .
+- Configuring BAM tools requires certain SQL Server administrative functionality and must be performed from a machine that has Integration Services installed. The version of Integration Services must be equivalent version as your target SQL Server.
+- Starting BizTalk Server 2020, BAM uses SSIS Catalog for deploying and managing BAM DTS packages (https://docs.microsoft.com/en-us/biztalk/core/bam-dts-packages). Create SSIS Catalog on the target SQL Server (Primary Import and Star Schema database) before configuration using https://docs.microsoft.com/en-us/sql/integration-services/catalog/ssis-catalog?view=sql-server-ver15#create-the-ssis-catalog 
```

Comment:
suggestion: Starting with **BizTalk Server 2020**

Reply:
would it be better to have some text and associated links rather than links in the text directly?
e.g. [BAM DTS packages](https://docs.microsoft.com/en-us/biztalk/core/bam-dts-packages)
and similarly for [Create SSIS Catalog](https://docs.microsoft.com/en-us/sql/integration-services/catalog/ssis-catalog?view=sql-server-ver15#create-the-ssis-catalog)

Reply:
also, links should not have the language identifier in the URL 'en-us'